lms/templates/instructor/instructor_dashboard_2/instructor_dashboard_2.html

```diff
@@ -116,11 +116,12 @@
         ## to keep this short, sections can be pulled out into their own files
 
         % for section_data in sections:
-          <section id="${ section_data['section_key'] }" class="idash-section">
-            <%include file="${ section_data['section_key'] }.html" args="section_data=section_data" />
+          <section id="${ section_data['section_key'] }" class="idash-section" aria-labelledby="header-${section_data['section_key']}">
```

Comment:
Yes, duh. We don't want to use prescribed "perfect" fixture markup. We want to use what's actually present in the real DOM. I'll take a look at the aXe documentation and see if this is something we can enable or possibly add.

The BokChoy test I wrote was moved to AC-415. I'd be curious to see what it finds here though, so maybe I'll move it back.